Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -26,6 +28,7 @@ public class TextDocumentLoader : ITextDocumentLoader {
     // 256MB
     private const int MaxStackSize = 0x10000000;
 
+    private DafnyOptions Options => DafnyOptions.O;
```

Comment:
Is the next step to make this a field?

Reply:
Seems like it, but I'd prefer doing that in a different PR